vpr/src/place/place.cpp

```diff
@@ -2631,8 +1905,7 @@ static void alloc_and_load_placement_structs(float place_cost_exp,
         }
     }
 
-    net_cost.resize(num_nets, -1.);
-    proposed_net_cost.resize(num_nets, -1.);
+    init_net_cost_structs(num_nets);
 
     if (cube_bb) {
```

Comment:
Refactor into an alloc_and_load_place_move_structs () pr init_place_move_structs. 
The code is very similar to init_try_swap_net_cost_structs but I think works on different variables.  Maybe the init_try_swap_net_cost_structs can be reused (would need to have more abstraction; if that seems messy then just make a new init_ or alloc_and_load_ routine.